packages/rn-tester/RNTester-visionOS/App.swift

```diff
@@ -5,11 +5,17 @@ import React_RCTSwiftExtensions
 @main
 struct RNTesterApp: App {
   @UIApplicationDelegateAdaptor var delegate: AppDelegate
+  @Environment(\.reactContext) private var reactContext
+  
   @State private var immersionLevel: ImmersionStyle = .full
   
   var body: some Scene {
     RCTMainWindow(moduleName: "RNTesterApp")
+    
+    RCTWindow(id: "SecondWindow", sceneData: reactContext.getSceneData(id: "SecondWindow"))
```

Comment:
[Question] How can we make declaring new windows possible without users touching native code?